src/tonic/api/collections_api.rs

```diff
@@ -308,6 +346,9 @@ impl Collections for CollectionsService {
         &self,
         mut request: Request<DeleteShardKeyRequest>,
     ) -> Result<Response<DeleteShardKeyResponse>, Status> {
+
+        validate(request.get_ref())?;
```

Comment:
error[E0277]: the trait bound `api::grpc::DeleteShardKeyRequest: validator::Validate` is not satisfied
   --> src/tonic/api/collections_api.rs:350:18
    |
350 |         validate(request.get_ref())?;
    |         -------- ^^^^^^^^^^^^^^^^^ the trait `validator::Validate` is not implemented for `api::grpc::DeleteShardKeyRequest`
    |         |
    |         required by a bound introduced by this call